### PR TITLE
chore(litmus-portal): Error page updated.

### LINKED ISSUE #2011: Error page update for litmus-portal.
Update 404 page as per new design for litmus-portal.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan.gupta@mayadata.io>

## Proposed changes

Error page refractor and update as per new designs.
fixes #2011 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- None.

## Special notes for your reviewer:
Needed for release 1.8.0

### CODE REVIEW

File: litmus-portal/frontend/src/pages/ErrorPage/styles.ts [hunk 1]
```diff
@@ -1,18 +1,148 @@
...
+    },
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '30px',
+    },
+  },
+  backBtn: {
+    width: 117,
+    height: 44,
+    padding: 15,
+    backgroundColor: '#5B44BA',
```
Comment (amityt) [inline litmus-portal/frontend/src/pages/ErrorPage/styles.ts:98]: The colors can be used from the theme.

File: litmus-portal/frontend/src/pages/ErrorPage/styles.ts [hunk 2]
```diff
@@ -1,18 +1,148 @@
...
+    textTransform: 'none',
+    color: '#FFFFFF',
```
Comment (amityt) [inline litmus-portal/frontend/src/pages/ErrorPage/styles.ts:100]: We can use theme.palette here

File: litmus-portal/frontend/src/pages/ErrorPage/styles.ts [hunk 3]
```diff
@@ -1,18 +1,148 @@
...
+      backgroundColor: '#5B44BA',
+    },
+    [theme.breakpoints.down('md')]: {
+      marginTop: 10,
+    },
+  },
+  descText: {
+    fontSize: '16px',
+    color: '#000000',
+    paddingTop: 20,
```
Comment (amityt) [inline litmus-portal/frontend/src/pages/ErrorPage/styles.ts:111]: Theme.spacing can be used here

File: litmus-portal/frontend/src/pages/ErrorPage/styles.ts [hunk 4]
```diff
@@ -1,18 +1,148 @@
...
+    paddingBottom: 20,
+    [theme.breakpoints.up('lg')]: {
+      fontSize: '30px',
+      paddingTop: 20,
+      paddingBottom: 20,
+    },
+  },
+  errImg: {
+    width: 420,
```
Comment (amityt) [inline litmus-portal/frontend/src/pages/ErrorPage/styles.ts:120]: Can we change the height and width in rem?

**Comment by ishangupta-ds** [comment]
@amityt PTAL. Changes have been made.

**Review by amityt — APPROVED** [review]

**Review by arkajyotiMukherjee — DISMISSED** [review]
LGTM

**Review by S-ayanide — APPROVED** [review]

**Comment by ishangupta-ds** [comment]
@rajdas98 PTAL.

**Review by amityt — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**